src/ux/progress.ts

```diff
@@ -86,12 +87,15 @@ export class Progress extends UxBase {
   }
 
   private _start(total: number, payload: Progress.Payload = {}): void {
+    if (this.started) return;
     const start = once((bar: Progress.Bar, t: number, p: Progress.Payload = {}) => {
```

Comment:
How's this change any different than using this `once()` function?
Should that already protect against multiple starts?

Reply:
@RodEsp good question. Turns out `once` here was pointless (as was my original solution) because the true problem was executing `this.bar = CliUx.ux.progress` every single time, regardless of it the `this.bar.start` was memoized or not.

So I updated my solution to check `this.started` further up in the `start` method. This will prevent any subsequent calls to `CliUx.ux.progress` if the bar has already been started. 

I also made a PR in plugin-deploy-retrieve to make it so that we don't even try to call `start` multiple times to begin with: https://github.com/salesforcecli/plugin-deploy-retrieve/pull/258

Thanks for making me take a closer look!

Reply:
Ah, yes. Good catch!